frontend/javascripts/test/api/api_skeleton_latest.spec.js

```diff
@@ -199,7 +199,7 @@ test("Utils Api: registerOverwrite should overwrite an existing function", t =>
 
 test("Calling a volume api function in a skeleton tracing should throw an error", t => {
   const api = t.context.api;
-  t.throws(() => api.tracing.getVolumeTool());
+  t.throws(() => api.tracing.getAnnotationTool());
```

Comment:
I would assume this test to fail, since `getAnnotationTool()` no longer only works in volume tracings. A different volume-only api method should be chosen for this test.